apps/varslinger-service/src/main/java/no/nav/registre/varslingerservice/controller/VarslingerController.java

```diff
@@ -27,12 +20,16 @@ public class VarslingerController {
 
     @GetMapping
     public ResponseEntity<List<VarslingDTO>> getVarslinger() {
-        var list = varslingerAdapter.getAll().stream().map(Varsling::toDTO).collect(Collectors.toList());
+        var list = varslingerAdapter
+                .getAll()
+                .stream()
+                .map(Varsling::toDTO)
+                .toList();
         return ResponseEntity.ok(list);
     }
 
     @PutMapping
-    public ResponseEntity<?> oppdaterVarslinger(@RequestBody VarslingDTO dto) {
+    public ResponseEntity<Object> oppdaterVarslinger(@RequestBody VarslingDTO dto) {
```

Comment:
```
@ResponseStatus(HttpStatus.CREATED)
public void oppdaterVarslinger(@RequestBody VarslingDTO dto) {
```

Reply:
Her blir det litt krøll, siden ResponseEntity brukes aktivt for å returnere 201 samt å sette en header "Location".

Reply:
Ja return type void var feil, skulle ha vært String, slik:
```
@ResponseStatus(HttpStatus.CREATED)
public String oppdaterVarslinger(@RequestBody VarslingDTO dto) {
```

Reply:
Da vil APIet endre oppførsel, i og med at response header `Location `ikke blir satt. Om det ikke er i bruk av konsumentene, så ja - absolutt. Men denne PRen ble allerede mer omfattende enn tiltenkt om man ikke samtidig skulle begynne å se på konsumentene også. 😄

Apropos, la til et par TODOs i koden som fortjener egne PRs.